### PR TITLE
[DON'T MERGE ⚠️ ] Create protected "Configuration" branch: announcements/strategy

### PR DESCRIPTION
**Pull request by W3stside**
Creates a protected `configuration` branch that supersedes `announcements` branch and adds a `price > strategy.json` fs to allow us and BE to change the primary/secondary price strategy

## Testing
Not applicable here, wait until develop/master updated to meet the new requirements set forth from this branch's changes

### CODE REVIEW

File: configuration/price/strategy.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (alfetopito) [inline configuration/price/strategy.json:1]: Shouldn't this be network based as well, like the announcements?

Comment (W3stside) [reply configuration/price/strategy.json:1]: i wwas thinking that it shouldn't actually. but maybe it doesn't hurt to include that 🤷

Comment (alfetopito) [reply configuration/price/strategy.json]: My thinking was, maybe only Mainnet endpoint has issues and GC is fine, so we wouldn't need to change the setting for both.

File: config/announcements/announcements.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Beware, *cowswap.sale* is **NOT IN ANY WAY** affiliated with [*cowswap.exchange*](https://cowswap.exchange). For more info check our [discord server](https://chat.cowswap.exchange).
```
Comment (alfetopito) [inline config/announcements/announcements.md:1]: Actually, you can remove this file, since it's chain based

**Comment by W3stside** [comment]
closing as agreed config

**Pull request closed without merging**